components/create_team/components/team_url/team_url.tsx

```diff
@@ -151,12 +162,12 @@ export default class TeamUrl extends React.PureComponent {
         }
     }
 
-    handleFocus = (e) => {
+    public handleFocus = (e: React.FocusEvent<HTMLInputElement>) => {
         e.preventDefault();
         e.currentTarget.select();
     }
 
-    render() {
+    public render() {
```

Comment:
Nit: This doesn't need to be `public`, we're never calling it outside of the scope of this class.